README.rst

```diff
@@ -101,7 +101,7 @@ rule of the top-level parser must be defined with an arbitrary combination of th
 Example usage to reduce an HTML file::
 
     picireny --input=<path/to/the/input.html> --test=<path/to/the/tester> \
-             --grammar "HTMLLexer.g4 HTMLParser.g4" --start htmlDocument \
+             --grammar "HTMLLexer.g4" "HTMLParser.g4" --start htmlDocument \
```

Comment:
Hm, that seems to be right, `--grammar` accepts multiple arguments (instead of a single argument with space-separated file names). But are the quotes then needed at all? They could (should) be removed from around the file names.